### PR TITLE
plat-ls: add grapeboard ls1012 platform flavor

### PR DESCRIPTION
**Pull request by jordanrh1**
Add support for Scalys Grapeboard (https://www.grapeboard.com/).

Tested-by: Jordan Rhee <jordanrh@microsoft.com>
Signed-off-by: Jordan Rhee <jordanrh@microsoft.com>

@evertpap Please take a look.

### CODE REVIEW

File: core/arch/arm/plat-ls/conf.mk [hunk 1]
```diff
@@ -38,6 +38,15 @@ $(call force,CFG_CORE_CLUSTER_SHIFT,2)
 CFG_SHMEM_SIZE ?= 0x00200000
 endif
 
+ifeq ($(PLATFORM_FLAVOR),ls1012grapeboard)
+CFG_HW_UNQ_KEY_REQUEST ?= y
+include core/arch/arm/cpu/cortex-armv8-0.mk
+$(call force,CFG_TEE_CORE_NB_CORE,1)
```
Comment (jordanrh1) [inline core/arch/arm/plat-ls/conf.mk]: This file should probably be refactored at some point to put soc-specific configuration in one place.

**Comment by jforissier** [comment]
LGTM. @pangupta?

**Comment by pangupta** [comment]
Sorry for the late response.

The code changes are fine. 
It can be merged.

**Comment by jforissier** [comment]
@jordanrh1 I think we can take @pangupta 's answer as an `Acked-by: Pankaj Gupta <pankaj.gupta@nxp.com>`, would you mind updating the patch accordingly? Thanks!

**Pull request closed without merging**